bin/prepare-release.sh

```diff
@@ -88,7 +88,7 @@ run rm "${WORKSPACE_ROOT}/${FILE}.bak"
 
 FILE=manifest.json
 run $SED -i.bak -E \
-    -e "s/^\"version\": \"[0-9a-z.-]+\"/\"version\": \"${NEW_VERSION}\"/" \
+    -e "s/\"version\": \"[0-9a-z.-]+\"/\"version\": \"${NEW_VERSION}\"/" \
```

Comment:
Off-topic, but something that really parses the JS like `jq` could do this search-and-replace.

Reply:
Good point. Let me file an issue for that!

Reply:
https://github.com/mozilla-rally/core-addon/issues/264